WhateverGreen/kern_igfx.cpp

```diff
@@ -198,6 +199,18 @@ void IGFX::processKernel(KernelPatcher &patcher, DeviceInfo *info) {
 		KernelPatcher::RouteRequest request("__ZN9IOService20copyExistingServicesEP12OSDictionaryjj", wrapCopyExistingServices, orgCopyExistingServices);
 		patcher.routeMultiple(KernelPatcher::KernelID, &request, 1);
 	}
+	
+	// Enable CFL backlight patch if CPU gen is CFL and model is laptop
+	cflBacklightPatch = (cpuGeneration == CPUInfo::CpuGeneration::CoffeeLake && WIOKit::getComputerModel() == WIOKit::ComputerModel::ComputerLaptop);
+	cflBacklightPatch = cflBacklightPatch || (info->videoBuiltin && info->videoBuiltin->getProperty("enable-cflbklt"));
```

Comment:
For the IORegistry property there is probably no good reason to shorten it way too much. What about renaming `enable-cflbklt` to `enable-cfl-backlight-fix` = `<01>`?